addons/web/static/src/core/color_picker/color_picker.xml

```diff
@@ -43,20 +43,63 @@
                 t-on-focusout="onColorHoverOut"
                 >
                 <!-- List all Presets -->
-                <t t-foreach="[1, 2, 3, 4, 5]" t-as="number" t-key="number">
-                    <t t-set="className" t-value="'o_cc' + number"/>
-                    <t t-set="activeClass" t-value="this.props.state.selectedColorCombination === className ? 'selected' : ''"/>
-                    <button
+                <t t-if="!props.themePresets">
+                    <t t-foreach="[1, 2, 3, 4, 5]" t-as="number" t-key="number">
+                        <t t-set="className" t-value="'o_cc' + number"/>
+                        <t t-set="activeClass" t-value="this.props.state.selectedColorCombination === className ? 'selected' : ''"/>
+                        <button
+                                type="button"
+                                class="w-100 p-0 border-0 bg-transparent p-2 d-flex justify-content-between align-items-center color-combination-button"
+                                t-att-class="[className, activeClass].join(' ')"
+                                t-att-data-color="className"
+                                t-attf-title="Preset {{number}}">
+                            <h1 class="m-0 fs-4">Title</h1>
+                            <p class="m-0 flex-grow-1">Text</p>
+                            <span class="py-1 px-1 rounded-1 fs-6 btn btn-sm lh-1 btn btn-sm me-1 d-flex flex-column justify-content-center btn-primary"><small>Button</small></span>
+                            <span class="py-1 px-1 rounded-1 fs-6 btn btn-sm lh-1 btn btn-sm d-flex flex-column justify-content-center btn-secondary"><small>Button</small></span>
+                        </button>
+                    </t>
+                </t>
+                <t t-else="">
```

Comment:
I don't think it makes sense to have the theme tab without preset so I won't make a "default preset".
What about
```xml
<t t-if="state.activeTab==='theme'">
  ...
  <t t-foreach="props.themePresets" t-as="preset" t-key="preset.id">
```
Let the code crash if there is no `themePresets` available.